pkg/pipelines/imagerepo/imagerepo.go

```diff
@@ -18,7 +18,8 @@ import (
 
 // ValidateImageRepo validates the input image repo.  It determines if it is
 // for internal registry and prepend internal registry hostname if necessary.
-func ValidateImageRepo(imageRepo, registryURL string) (bool, string, error) {
+func ValidateImageRepo(imageRepo string) (bool, string, error) {
+	registryURL := "image-registry.openshift-image-registry.svc:5000"
```

Comment:
This could be moved to a top-level constant for visibility.

Reply:
make the required changes